### PR TITLE
#725 Qulice should use source level 1.7

### PR DESCRIPTION
**Pull request by dskalenko**
Qulice should use source level 1.7 #725

### CODE REVIEW

**Comment by dskalenko** [comment]
I am partially done second point : "2. Replace are cases where AutoCloseable is used with proper try-with-resources (in DataSourceReader, FindBugsValidator, PMDValidatorTest, Prettifier)"
because for  DataSourceReader (this one not used at all, may we have to remove) and PMDValidatorTest replacement not possible

**Comment by davvd** [comment]
@dskalenko Many thanks for the PR, let me find a reviewer for it

**Comment by davvd** [comment]
@mkordas please review, thanks

**Comment by mkordas** [comment]
@dskalenko I'm on it

**Comment by mkordas** [comment]
@dskalenko 
> PMDValidatorTest replacement not possible

Why this replacement is not possible?

File: qulice-maven-plugin/pom.xml [hunk 1]
```diff
@@ -210,7 +210,7 @@
         <dependency>
             <groupId>org.sonatype.sisu</groupId>
             <artifactId>sisu-inject-plexus</artifactId>
-            <version>2.5.3</version>
+            <version>2.6.0</version>
```
Comment (mkordas) [inline qulice-maven-plugin/pom.xml]: @dskalenko is it in scope of this PR? why the change was required?

**Comment by mkordas** [comment]
@dskalenko don't we have any try with manual `close()` in finally that could be replaced with try-with resources?

**Comment by mkordas** [comment]
@dskalenko see my comments above. Is the PR fully ready for a review now?

File: qulice-maven-plugin/pom.xml (hunk 1 above)
Comment (dskalenko) [inline qulice-maven-plugin/pom.xml]: @mkordas 
it because when I changed source level to 1.7 I got this error http://stackoverflow.com/questions/10738609/what-does-supported-source-version-release-5-from-annotation-processor-me

**Comment by dskalenko** [comment]
@mkordas 
> PMDValidatorTest replacement not possible
Why this replacement is not possible?

because I didn't find using of AutoCloseable resource

**Comment by dskalenko** [comment]
@mkordas 

> don't we have any try with manual close() in finally that could be replaced with try-with resources?

I just tried replace only classes which were requested in issue

File: qulice-maven-plugin/pom.xml (hunk 1 above)
Comment (mkordas) [inline qulice-maven-plugin/pom.xml]: @dskalenko thanks!

**Comment by mkordas** [comment]
@dskalenko OK, so now it's up to @krzyk to verify his issue :)

**Comment by mkordas** [comment]
@rultor merge

**Comment by krzyk** [comment]
@dskalenko looks good, thanks

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by mkordas** [comment]
@rultor deploy

**Comment by krzyk** [comment]
@rultor deploy

**Comment by davvd** [comment]
@mkordas 10 mins added to @krzyk account (our architect), payment ID is `80637266`, since quality is good here

**24 mins** added to your account (payment number `56ea67e3f46357006f0002c7`), many thanks for your contribution! 46 hours and 20 mins spent here.

extra minutes for review comments (c=9)

+24 to your rating, your total score is [+8920](http://www.netbout.com/b/36080?open=rating)

**Comment by davvd** [comment]
@rultor deploy pls
